### ISSUE TITLE
Feature request: specific file-save only linters

### ISSUE THREAD

**Issue by oblitum** [iamcco/coc-diagnostic#74]
I wished I could set some specific linter to run just on file-save. Coc is currently in default mode (diagnostic on insert leave), which is fine for all language servers, and some light linters of coc-diagnostic, but for example, the textidote tool takes too much processing to check Markdown/LaTeX, so I wished just this one to run on file-save only.

**Comment by iamcco** [comment]
Make senses

**Comment by dpriedel** [comment]
I have an LSP server module front end to LanguageTool at: https://github.com/dpriedel/languagetool_languageserver.

It runs LanguageTool in http mode on the local machine.

It is a Python program which only requires a recent version of pygls.

I do not have an installer for it but it is just 1 module.

I also have a coc.nvim extension at: https://github.com/dpriedel/coc_ltls

Also, no installer -- I don't work with JS applications much.  It works on file open and save.
It can be configured using CocConfig.  If you have coc.json extension, you can get prompts
for ltls. settings.

I've been using it while I edit my wife's novel and it works fine for me.

Let me know if you have questions.

Dave Riedel

**Comment by oblitum** [comment]
@dpriedel that's nice, but, I wished that for [TeXtidote](https://sylvainhalle.github.io/textidote/), which better wraps Language Tool for LaTeX/Markdown authoring.

**Comment by dpriedel** [comment]
Yes it does but I'm using minimal LaTeX markup so I'm OK with a simpler interface

**Comment by oblitum** [comment]
@dpriedel fyi, I just published [coc-ltex](https://www.npmjs.com/package/coc-ltex), you can simply `:CocInstall` it :) It's much better than TeXtidote or raw Language Tool.

**Comment by sisrfeng** [comment]
Any progress ?
